lib/SILOptimizer/Mandatory/TFCanonicalizeCFG.cpp

```diff
@@ -19,14 +19,27 @@
 //===----------------------------------------------------------------------===//
 
 #include "TFCanonicalizeCFG.h"
+#include "TFUtilities.h"
+#include "llvm/ADT/iterator_range.h"
```

Comment:
Please move the llvm include below the swift includes since LLVM is a lower layer than swift.